test/univariate_statistics_test.cpp

```diff
@@ -924,200 +924,202 @@ void test_mode(ExecutionPolicy&& exec)
     test_integer_gini_coefficient<int>(std::execution::par);
-    test_integer_gini_coefficient<int>(std::execution::par_unseq);
+    //test_integer_gini_coefficient<int>(std::execution::par_unseq);
 
     test_sample_gini_coefficient<float>(std::execution::seq);
     test_sample_gini_coefficient<float>(std::execution::par);
-    test_sample_gini_coefficient<float>(std::execution::par_unseq);
+    //test_sample_gini_coefficient<float>(std::execution::par_unseq);
     test_sample_gini_coefficient<double>(std::execution::seq);
     test_sample_gini_coefficient<double>(std::execution::par);
-    test_sample_gini_coefficient<double>(std::execution::par_unseq);
+    //test_sample_gini_coefficient<double>(std::execution::par_unseq);
 
     test_sample_gini_coefficient<long double>(std::execution::seq);
     test_sample_gini_coefficient<long double>(std::execution::par);
-    test_sample_gini_coefficient<long double>(std::execution::par_unseq);
+    //test_sample_gini_coefficient<long double>(std::execution::par_unseq);
     test_sample_gini_coefficient<cpp_bin_float_50>(std::execution::seq);
     test_sample_gini_coefficient<cpp_bin_float_50>(std::execution::par);
-    test_sample_gini_coefficient<cpp_bin_float_50>(std::execution::par_unseq);
+    //test_sample_gini_coefficient<cpp_bin_float_50>(std::execution::par_unseq);
 
     test_interquartile_range<double>(std::execution::seq);
     test_interquartile_range<double>(std::execution::par);
-    test_interquartile_range<double>(std::execution::par_unseq);
+    //test_interquartile_range<double>(std::execution::par_unseq);
     test_interquartile_range<cpp_bin_float_50>(std::execution::seq);
     test_interquartile_range<cpp_bin_float_50>(std::execution::par);
-    test_interquartile_range<cpp_bin_float_50>(std::execution::par_unseq);
+    //test_interquartile_range<cpp_bin_float_50>(std::execution::par_unseq);
 
     test_integer_mode<int>(std::execution::seq);
     test_integer_mode<int>(std::execution::par);
-    test_integer_mode<int>(std::execution::par_unseq);
+    //test_integer_mode<int>(std::execution::par_unseq);
     test_integer_mode<int32_t>(std::execution::seq);
     test_integer_mode<int32_t>(std::execution::par);
-    test_integer_mode<int32_t>(std::execution::par_unseq);
+    //test_integer_mode<int32_t>(std::execution::par_unseq);
     test_integer_mode<int64_t>(std::execution::seq);
     test_integer_mode<int64_t>(std::execution::par);
-    test_integer_mode<int64_t>(std::execution::par_unseq);
+    //test_integer_mode<int64_t>(std::execution::par_unseq);
     test_integer_mode<uint32_t>(std::execution::seq);
     test_integer_mode<uint32_t>(std::execution::par);
-    test_integer_mode<uint32_t>(std::execution::par_unseq);
+    //test_integer_mode<uint32_t>(std::execution::par_unseq);
 
     test_mode<float>(std::execution::seq);
     test_mode<float>(std::execution::par);
-    test_mode<float>(std::execution::par_unseq);
+    //test_mode<float>(std::execution::par_unseq);
```

Comment:
Why are the `par_unseq` commented out?

Reply:
The only remaining CI failure was `C1128: number of sections exceeded object file format limit: compile with /bigobj`. I commented out all of the std::execution::par_unseq tests to drop the size by 1/3. The only implementations that vary based on par vs par_unseq are STL algorithms.

Reply:
Ok, I'm cool with that. Want to just delete this rather than comment it out?

Reply:
I have deleted them.